ci/bin/fly_sandbox.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Would be better renamed as `fly_pipeline` or similar (and update comment) as now flies to any environment depending on the `ENVIRONMENT` variable.

Reply:
I agree. Done.